xterm/config

```diff
@@ -1 +1,2 @@
 base_port=555
+flavors=1
```

Comment:
Shouldn't this be flavors=0

Reply:
If you want to keep it off for root capable users too, then yes? But I thought we could keep it off only for domain owners?

Reply:
Okay, fixed it to be off by default.